website/static/js/comment.js

```diff
@@ -291,6 +511,50 @@ var CommentModel = function(data, $parent, $root) {
         self.author = self.$root.author;
     }
 
+    self.editableContent = ko.computed(function() {
+        var content = self.content();
+        if (!content) {
+            content = '';
+        }
+        var regex = /\[(@|\+)(.*?)\]\(\/([a-z\d]{5})\/\)/;
+        var matches = content.match(/\[(@|\+)(.*?)\]\(\/([a-z\d]{5})\/\)/g);
```

Comment:
Okay. We should at least be able to factor out the pattern itself, yes? Something like (untested) ...

```javascript
var pattern = 'foo',
    bar = new RegEx(pattern),
    baz = new RegEx(pattern, 'g');
```